xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3602,11 +3602,7 @@ bool CVideoPlayer::OpenVideoStream(CDVDStreamInfo& hint, bool reset)
 
     // look for any EDL files
     m_Edl.Clear();
-    if (hint.fpsrate > 0 && hint.fpsscale > 0)
-    {
-      float fFramesPerSecond = (float)m_CurrentVideo.hint.fpsrate / (float)m_CurrentVideo.hint.fpsscale;
-      m_Edl.ReadEditDecisionLists(m_item, fFramesPerSecond, m_CurrentVideo.hint.height);
-    }
+    m_Edl.ReadEditDecisionLists(m_item);
```

Comment:
I wouldn't have removed fFramesPerSecond completely, just set it to zero if not supplied. Then fail in ReadEdl only if framerate required and not supplied. Containers like mkv do supply framerate correctly.

Reply:
Ok

Reply:
But that leaves the hack in `ReadEditDecisoonLists` around frames per second then. Why not completely clean it up?

Reply:
BTW. I have another version ready of it just being set to zero. As I was doing It was thinking if these Comskip and EDL files are rare they would be rare for every format not just TS.

Reply:
> But that leaves the hack in ReadEditDecisoonLists around frames per second then.

Wrong. The hack is for mpeg-ts. Containers like mkv provide correct framerate.

Reply:
Ok, I get you now. Will update with the other version that just removes the hack. Thanks again for your help.

Reply:
So the signature will pass the frame rate but no longer the height.